catalog/spatial/ogc/spatial-ogc-common/src/main/java/org/codice/ddf/spatial/ogc/catalog/common/converter/XmlNode.java

```diff
@@ -119,12 +123,17 @@ private String reconstructNode() {
 
         }
         reader.moveUp();
-        return buffer.toString();
+        node = buffer.toString();
+        return node;
     }
 
     @Override
     public String toString() {
-        return reconstructNode();
+        if (reader != null) {
```

Comment:
I would flip this conditional to:
```
if (reader == null){
   return super.toString();
} else {
  return reconstructNode();
}
```

Might also consider:
`return (reader == null) ? super.toString() : reconstructNode();`